index.markdown

```diff
@@ -472,6 +472,18 @@ are likely to just disappoint the winning team. When organizations/individuals
 wish to pursue further development of a hack they should speak to the winning
 team separately.
 
+### Attitude to prize winners
+
+Those who have won a prize at a hack day should not be expected to provide any
+ongoing service or favours to sponsors or organisers. Some hack day sponsors
```

Comment:
I like the sentiment, but at the same time there's things like BattleHack, where the prize _is_ to be flown to a finals, and we did expect people to at least allow us to talk about them.